torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -24,6 +24,8 @@ namespace torch {
 namespace jit {
 namespace tensorexpr {
 
+std::unordered_map<const Var*, const For*> LoopNest::index_loop_mapping;
```

Comment:
I can't think of a good reason this should be a global variable.  Can't it be a member of LoopNest?  Or better yet a local variable in the appropriate simplification method?